operations/kubernetes/Kyverno.md

```diff
@@ -456,3 +456,20 @@ and [match/exclude](https://release-1-10-0.kyverno.io/docs/writing-policies/matc
 
     If the issue still persists after restarting the Kyverno pods, then HPE recommends disabling the background scanning in the Kyverno policy.
     Set `background` to `false` to disable background scanning in the Kyverno policy.
+* Multiple Kyverno admission reports may cause etcd downtime during upgrade to CSM 1.6.
```

Comment:
This states that the problem *may* happen. I think an administrator reading this would reasonably wonder if this means that they should proactively do this procedure to avoid the problem.

The text doesn't convey whether this is a common problem (where it would then make sense to do this proactively) or if it is a rare problem (where it would then perhaps make sense to only worry about this if they hit it).

(From comment on original PR: https://github.com/Cray-HPE/docs-csm/pull/6178#discussion_r2248267925)